clearblade.md

```diff
@@ -1407,12 +1407,31 @@ Please use the APIs listed here - https://docs.couchdb.org/en/stable/api/index.h
 	db.performOperation(callback, httpMethod, uri, JSON.stringify(data))
 ~~~~
 
+### BigQuery 
+
+- arguments:
+  - operation
```

Comment:
do we only support only query operation? Is there a place people should refer to for more?